package.json

```diff
@@ -10,6 +10,8 @@
     "@types/react-dom": "^17.0.11",
     "@walletconnect/ethereum-provider": "^1.7.5",
     "ethers": "^5.5.4",
+    "google-protobuf": "^3.20.0-rc.2",
+    "grpc-web": "^1.3.1",
```

Comment:
For what are `google-protobuf` and `grpc-web` dependencies used? I'm just curious about them as this is new for me

Reply:
For runtime. They are used by the generated JS.